### PR TITLE
ci: Add new owners

### PR DESCRIPTION
**Pull request by mateusoliveira43**
Add new owners.

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -9,9 +9,17 @@ approvers:
...
 reviewers:
   - sseago
   - shubham-pampattiwar
   - kaovilai
   - eemcmullan
   - savitharaghunathan
+  - mateusoliveira43
+  - mrnold
+  - mpryc
+  - weshayutin
```
Comment (mateusoliveira43) [inline OWNERS:25]: Did I forget any one? There is also an OWNERS_ALIASES file, need to update that as well?

**Review by kaovilai — APPROVED** [review]

**Comment by weshayutin** [comment]
/hold 
for discussion

File: OWNERS (hunk 1 above)
Comment (kaovilai) [reply OWNERS:25]: unless you have or will change username, this should be ok as is.. perhaps also discuss if someone would like to be removed to not be pinged about this.

**Comment by mateusoliveira43** [comment]
Created issue about it https://github.com/openshift/oadp-operator/issues/1258
Please do not reopen this PR

**Pull request closed without merging**